curation/sequence_services.py

```diff
@@ -0,0 +1,33 @@
+"""Provide sequence ID generation services."""
+from biocommons.seqrepo import SeqRepo
+from os import environ
+from pathlib import Path
+import logging
+from curation import SEQREPO_DATA_PATH
+
+
+logger = logging.getLogger('curation_backend')
+logger.setLevel(logging.DEBUG)
+
+
+def get_seqrepo() -> SeqRepo:
+    """Instantiate SeqRepo instance.
+    :return: SeqRepo instance
+    """
+    seqrepo_path = Path(SEQREPO_DATA_PATH)
+    if not seqrepo_path.exists():
+        raise NotADirectoryError(f'Invalid SeqRepo path provided at '
+                                 f'environment variable SEQREPO_PATH: '
+                                 f'{seqrepo_path}')
+    return SeqRepo(seqrepo_path)
+
+
+sr = get_seqrepo()
+
+
+def get_ga4gh_sequence_id(sequence: str) -> str:
+    """Get GA4GH sequence ID for a given sequence.
+    :param str sequence: user-provided sequence name
+    :return: Dict containing `sequence_id` and `warnings` fields
+    """
+    return sr.translate_identifier(sequence, 'ga4gh')[0]
```

Comment:
Would it be best to capture errors in this method rather than views?

Reply:
I was planning on keeping request validation and error handling in views for the sake of something like balance (otherwise the views end up being pretty empty) but I am... reconsidering

Reply:
Will handle in #13 with more exhaustive error handling (do things like check for correct input structure in the views and content validation like checking for the existence of the sequence within the service modules)